airflow/utils/state.py

```diff
@@ -112,17 +112,17 @@ class State:
         TaskInstanceState.RUNNING: 'lime',
         TaskInstanceState.SUCCESS: 'green',
         TaskInstanceState.SHUTDOWN: 'blue',
-        TaskInstanceState.RESTARTING: 'violet',
+        TaskInstanceState.RESTARTING: 'mediumpurple',
```

Comment:
wait, I don't think restarting and deferred/sensing are supposed to be the same

Reply:
You are correct, find-and-replace error.